Citrix/citrix-adc/ingest/parser.yml

```diff
@@ -264,3 +275,18 @@ stages:
       - set:
           event.reason: "{{ parse_audit_sslvpn_logs.message.reason }}"
         filter: "{{ parse_audit_sslvpn_logs.message.get('reason') != None }}"
+
+  set_bash_log_fields:
+    actions:
+      - set:
+          event.action: "execute_shell_command"
+          event.category: ["configuration"]
+          event.dataset: "audit_cmd"
+          event.type: ["change"]
+          process.command_line: "{{ parse_audit_bash.message.command }}"
+          process.name: "bash"
+          user.name: "{{ parse_audit_bash.message.source }}"
+
+      - set:
+          process.working_directory: "{{ parse_audit_bash.message.path }}"
+        filter: "{{ parse_audit_bash.message.path != '(null)' }}"
```

Comment:
According to the sample, the captured value is not a path but the pseudo-tty device (see [wikipedia](https://en.wikipedia.org/wiki/Devpts)). Except setting [`process.interactive = True`](https://www.elastic.co/guide/en/ecs/8.11/ecs-process.html#field-process-interactive) and a custom field, IMHO, it's not suitable in ECS fields.